src/transformers/models/auto/modeling_auto.py

```diff
@@ -309,6 +310,7 @@
         ),
         ("swin", "SwinForImageClassification"),
         ("convnext", "ConvNextForImageClassification"),
+        ("cvt", "CvtForImageClassification"),
```

Comment:
You can also add CvT to the feature_extraction_auto.py file, such that people will be able to do:

```
from transformers import AutoFeatureExtractor

feature_extractor = AutoFeatureExtractor.from_pretrained("microsoft/cvt-...")
```
You can map "cvt" to `ViTFeatureExtractor`.

Reply:
Done